gym_pybullet_drones/envs/BaseAviary.py

```diff
@@ -127,6 +132,26 @@ def __init__(
         self.SIM_FREQ = freq
         self.TIMESTEP = 1. / self.SIM_FREQ
         self.AGGR_PHY_STEPS = aggregate_phy_steps
+        # Constants for WIND_AERO_DRAG
+        self.rho = 1.225 #Density of air [kg/m^3]
```

Comment:
Are these constants specific to X500? I suppose this is fine since we only work with X500.

Reply:
The constants come from
Craig, William, Derrick Yeo, and Derek A. Paley. "Geometric attitude and position control of a quadrotor in wind." Journal of Guidance, Control, and Dynamics 43.5 (2020): 870-883.
"210 mm carbon fiber frame with a Matek F405 STD flight controller and Matek FCHUB-6S power distribution board. Gemfan 5030 propellers are mounted to EMAX RS-2205 motors that are controlled by EMAX Lightning 20A ESCs."
Which has a roughly similar form factor to the X500. Some constants will remain the same for x500, several will change, regardless it should be verified experimentally. These constants should be sufficient as a proof of concept of training in wind.